src/components/atoms/PortalVisibility/PortalVisibility.tsx

```diff
@@ -40,46 +39,29 @@ export const PortalVisibility: React.FC<PortalVisibilityProps> = ({
 
   const renderedItems = useMemo(
     () =>
-      Object.values(LABEL_VISIBILITY_OPTIONS).map(
-        ({ subtitle, label, value }) => {
-          const isSelected = isDefined(selected) && selected === value;
-          const itemClasses = classNames({
-            "PortalVisibility__item PortalVisibility__item--selected": isSelected,
-            "PortalVisibility__item PortalVisibility__item--unselected": !isSelected,
-          });
+      Object.values(LABEL_VISIBILITY_OPTIONS).map(({ label, value }) => {
+        const isSelected = isDefined(selected) && selected === value;
+        const imageClasses = classNames({
+          "portalImage bg-indigo-600 border-transparent text-white hover:bg-indigo-700 border rounded-md py-3 px-3 flex items-center justify-center text-sm font-medium uppercase sm:flex-1 bg-white border-gray-200 text-gray-900 hover:bg-white": isSelected,
+          "portalImage bg-white border-gray-200 text-gray-900 hover:bg-gray-50 border rounded-md py-3 px-3 flex items-center justify-center text-sm font-medium uppercase sm:flex-1 bg-white border-gray-200 text-gray-900 hover:bg-white": !isSelected,
+        });
 
-          return (
-            <div
-              key={label}
-              onClick={(e) => {
-                e.preventDefault();
-                setSelected(value);
-                setValue(name, value, { shouldValidate: true });
-              }}
-              className={itemClasses}
-            >
-              <div className="PortalVisibility__image">
-                {subtitle && (
-                  <div className="PortalVisibility__subtitle">
-                    {subtitle.map(({ text, icon }, index) => (
-                      <div
-                        key={`${text}-${index}`}
-                        className="PortalVisibility__subtitle-item"
-                      >
-                        <FontAwesomeIcon icon={icon} />
-                        <span className="PortalVisibility__subtitle-text">
-                          {text}
-                        </span>
-                      </div>
-                    ))}
-                  </div>
-                )}
-              </div>
-              <span className="PortalVisibility__caption">{label}</span>
+        return (
+          <div
+            key={label}
+            onClick={(e) => {
+              e.preventDefault();
+              setSelected(value);
+              setValue(name, value, { shouldValidate: true });
+            }}
+          >
+            <div className={imageClasses}>
+              <img src="#" alt="" />
```

Comment:
[Q]
Is this supposed to be a placeholder to be implemented in the future?
Maybe a `@debt` or `TODO` comment is needed.
Otherwise, what's it's purpose?

Reply:
added a debt comment [3469e91](https://github.com/sparkletown/sparkle/pull/2944/commits/3469e91693f840fa4d04232bc8c99b452b8f6b6d)